app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -88,8 +88,11 @@ def identity_needs_verification?
       ((@authorize_form.ial2_requested? || @authorize_form.ial2_strict_requested?) &&
         (current_user.decorate.identity_not_verified? ||
         decorated_session.requested_more_recent_verification?)) ||
-        (@authorize_form.ial2_strict_requested? &&
-        !current_user.active_profile&.includes_liveness_check?)
+        identity_needs_strict_ial2_verification?
+    end
+
+    def identity_needs_strict_ial2_verification?
```

Comment:
As I've recently discovered (LG-6217), we're already not very consistent, but: Should we have equivalent logic for SAML as well?

Reply:
Yes, however we don't support strict IAL2 in SAML today. I was going to figure out what it all looks like before moving it over

Reply:
Hm, now I'm wondering if the changes in #6253 shouldn't have been implemented, then? I seem to recall there were some failing specs that it was resolving from #6229, as if we had specs running through a SAML + IAL2Strict scenario.

Reply:
Interesting, I did not think SAML supported it at all. You may have jumped ahead of me a bit. Fortunately IAL2 strict is disabled in all the places that matter.